2017-08-05 15:48:58,234 udpserver.py[line:109] error_code:106_udp_third_not_in_11
2017-08-05 15:48:58,234 udpserver.py[line:109] error_code:106_udp_third_not_in_11
2017-08-05 15:48:58,234 udpserver.py[line:109] error_code:106_udp_third_not_in_11
2017-08-05 15:49:22,605 udpserver.py[line:109] error_code:106_udp_third_not_in_11
2017-08-05 15:49:22,605 udpserver.py[line:109] error_code:106_udp_third_not_in_11
2017-08-05 15:49:22,605 udpserver.py[line:109] error_code:106_udp_third_not_in_11
2017-08-05 15:50:05,375 udpserver.py[line:216] error_code:109_udp_third_life==0,delete,Ip_list=1
2017-08-05 15:50:05,375 udpserver.py[line:216] error_code:109_udp_third_life==0,delete,Ip_list=1
2017-08-05 15:50:05,375 udpserver.py[line:216] error_code:109_udp_third_life==0,delete,Ip_list=1
2017-08-05 15:50:52,776 udpserver.py[line:216] error_code:109_udp_third_life==0,delete,Ip_list=1
2017-08-05 15:50:52,776 udpserver.py[line:216] error_code:109_udp_third_life==0,delete,Ip_list=1
2017-08-05 15:50:52,776 udpserver.py[line:216] error_code:109_udp_third_life==0,delete,Ip_list=1
